sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -171,25 +171,20 @@ public static SentryEnvelopeItem fromAttachment(
         new CachedItem(
             () -> {
               if (attachment.getBytes() != null) {
-                if (attachment.getBytes().length > maxAttachmentSize) {
-                  throw new SentryEnvelopeException(
-                      String.format(
-                          "Dropping attachment with filename '%s', because the "
-                              + "size of the passed bytes with %d bytes is bigger "
-                              + "than the maximum allowed attachment size of "
-                              + "%d bytes.",
-                          attachment.getFilename(),
-                          attachment.getBytes().length,
-                          maxAttachmentSize));
-                }
+                ensureAttachmentSizeLimit(
```

Comment:
`l`: I think we could extract `attachment.getBytes()` into a variable and then reuse it when ensuring the size limit and returning, but it's a micro-optimization